js/add-field.js

```diff
@@ -167,19 +171,81 @@
   }
 
   // Create the asurite id checkboxes.
-  function update_tree() {
+  function update_tree(departments, size, page) {
     var departments = $(".directory-tree").val().split(',');
     //  var departments = ['1661','1374'];
-    var query = createCallParams(departments);
+    var query = createCallParams(departments, size || 100, page);
     $.getJSON("/endpoint/people-in-department" + query, function (json) {
       // Get existing data.
+      totalPages = json?.meta?.page?.total_pages;
```

Comment:
@davidornelas11 In JS that doesn't get packaged and polyfilled for older browser support, we need to be careful about using advanced JS, such as this that gets used in the CMS.

i'm going to say let's leave this, though, as it's on the Admin backend, and WS2 admins should be using modern browsers.  Just something to be aware of for the future, when writing JS for the CMS which doesn't get packaged by Webpack.